dockerplugin.py

```diff
@@ -567,8 +570,32 @@ def init_callback(self):
                            timeout=self.timeout))
         return True
 
+    # Method to compare docker version with min version required
+    def check_version(self, version):
+        if StrictVersion(version) < \
+                StrictVersion(DockerPlugin.MIN_DOCKER_API_VERSION):
```

Comment:
Minor: lets set `DockerPlugin.MIN_DOCKER_API_VERSION` to `StrictVersion('1.17')`, no need to it on each read.